### ISSUE TITLE
When will Windows get the latest release?

### ISSUE THREAD

**Issue by alanpep** [overviewer/Minecraft-Overviewer#1390]
All other Operating Systems have a release for v0.12.197, but Windows is still on v0.12.137.  When will this be updated?

**Comment by eminence** [comment]
Sorry about this, we're working on it.  In the meantime, you can get the latest Windows binaries from here : http://static.overviewer.org/builds/master/801/

**Comment by mgrotke** [comment]
Hi I was trying to follow the compile instructions (http://docs.overviewer.org/en/latest/building/), but I simply can't do it.  I think the instructions are missing a huge step.  The instructions say:

Prerequisites:  You will need a copy of the PIL sources.  (ok, check)
Building with Visual Studio:
Step 1:  Get the latest Overviewer source code (ok, check)
Step 2:  Open the Visual Studio 2010 Command Prompt (ok, check)
Step 3:  CD to the folder containing the Overviewer source code (ok, check)
Step 4:  Copy Imaging.h and ImPlatform.h from PIL to the current working directory (ok, check)
Step 5:  Try a build: "c:\python26\python setup.py build" (uh oh!)

I don't have python installed, never used it, and don't know how to use it.  This is missing in the instructions.  Sorry I'm dumb, but please help!  Also, these instructions are supposed to output a .pyd file, but I need an .exe.  I think the Overviewer build instruction page could use a little love :)

**Comment by CounterPillow** [comment]
Those instructions need to be updated considering it's python27 now. And yeah, it should mention that it requires python2 to be installed.

>Also, these instructions are supposed to output a .pyd file, but I need an .exe.

the .exe is just a py2exe thing which packages in a python version with overviewer so it can be redistributed easily. If you have python installed, you don't need a .exe

**Comment by jbjanot** [comment]
Hi!

Do you have Any ETA for this issue ?

Bye the way thank you very much for this amazing software!
Have a nice day.

**Comment by CounterPillow** [comment]
@jbjanot no ETA because nobody is paid to work on Overviewer, so things get done when somebody has free time and feels like spending it working on Overviewer

**Comment by jbjanot** [comment]
I understand of course, and thank you for the time you spend on this.
Would you accept any help in case I could figure out how to compile it
myself?

Le 8 juil. 2017 12:16 PM, "Nicolas F." <notifications@github.com> a écrit :

@jbjanot <https://github.com/jbjanot> no ETA because nobody is paid to work
on Overviewer, so things get done when somebody has free time and feels
like spending it working on Overviewer

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/overviewer/Minecraft-Overviewer/issues/1390#issuecomment-313847258>,
or mute the thread

**Comment by alandamore** [comment]
Firstly well done to the overviewer volunteers, some comments:
* The version win64-0.12.196 from the above link works for me on my MC 1.21.1 server files;
* I tried building on Windows 7x64. The python extension command is incorrect, "c:\python27\python.exe -m pip -U numpy pillow" should be "c:\python27\python.exe -m pip install numpy pillow"
* I went the "mingw-w64 and msys2" route, ultimately failed, when running the overviewer.py in minigw64 it fails with "ImportError: The _imaging extension was built for another version of Pillow or PIL: Core Version: "4.2.1"Pillow Version:  4.2.1". Due to lack of time and brainpower I gave up at this point and downloaded the .exe which seems fine. But if above contributers have managed to compile on Win 7 I'm happy to keep trying another method... Thanks for listening! Andy
